src/components/ArrayInput/index.ts

```diff
@@ -1,6 +1,7 @@
 import * as utils from '../../helpers/utils';
 
 import Element, { ElementArgs, IBindable, IBindableArgs, IFocusable } from '../Element';
+import { Observer } from '@playcanvas/observer';
```

Comment:
Minor point but as a convention, I have been importing external dependencies before everything else. So I'd move this to the top of the file.